float64 r_ang_1
float64 r_ang_2
float64 r_ang_3
float64 r_ang_4
float64 r_ang_5
float64 r_ang_6
float64 r_ang_7
float64 l_ang_1
float64 l_ang_2
float64 l_ang_3
float64 l_ang_4
float64 l_ang_5
float64 l_ang_6
float64 l_ang_7

